docker-compose.yml

```diff
@@ -38,3 +38,5 @@ services:
       POSTGRES_DB: cfp_v3
       POSTGRES_USER: cfp_v3
       POSTGRES_PASSWORD: cfp_v3
+    ports:
+    - "5432:5432"
```

Comment:
big thumbs up for adding quotes! not a problem here, but yaml is very touchy in some places (see https://docs.docker.com/compose/compose-file/compose-file-v3/#ports for more details, not a problem here, but still, it's a good practise).